web-components/src/components/icons/ShareIcons.tsx

```diff
@@ -70,17 +71,30 @@ export default function ShareIcons({
 
   return (
     <>
-      <Grid container className={classes.root} spacing={4}>
+      <Grid
+        container
+        className={classes.root}
+        spacing={2}
+        sx={{
+          [theme.breakpoints.up('md')]: {
+            gap: theme.spacing(4),
+          },
+        }}
```

Comment:
can we use tailwind instead?

Reply:
It was more convenient to do it this way because some Tailwind classes don't work with `preflight` disabled. However, I have now implemented those classes - see update in the `tailwind.css` file.

Reply:
I see
FYI `spacing` prop can also take responsive values, eg `spacing={{ xs: 1, sm: 2, md: 3 }}` but I guess it's probably best to work with tailwind anyway